plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -494,13 +514,16 @@ func (c *VolumeZoneChecker) predicate(pod *v1.Pod, meta interface{}, nodeInfo *s
 //     C1:
 //       CPU: 2
 //       Memory: 1G
+//       nvidia-gpu: 2
+//       solarflare-40gig: 1
 //     C2:
 //       CPU: 1
 //       Memory: 1G
 //
-// Result: CPU: 3, Memory: 3G
-func GetResourceRequest(pod *v1.Pod) *schedulercache.Resource {
+// Result: CPU: 3, Memory: 3G, ['nvidia-gpu': 2, 'solarflare-40gig': 1]
```

Comment:
This seems specific to your environment. What's the plan for making this more generic?